base/setup/usetup/consup.c

```diff
@@ -750,6 +832,51 @@ CONSOLE_SetStyledText(
     }
 }
 
+/**
+ * @CONSOLE_SetStyledTextByID
+ *
+ * Prints a text to the output - screen.
+ *
+ * @param[in]   page
+ *     The MUI (Multilingual User Interface) entry page number, as a unsigned long integer.
+ *
+ * @param[in]   TextID
+ *      The text identification number (ID), as a unsigned integer.
+ *
+ * @param[in]   Flags
+ *      The text style flag, as a unsigned integer.
+ *
+ * @return
+ *     Nothing.
+ *
+ * @remarks
+ *      Unlike CONSOLE_SetStyledText, CONSOLE_SetStyledTextByID outputs a portion of text given its text ID,
+ *      entry page interface and the specific style flag of the text. The routine calls MUIGetEntryData() to automatically
```

Comment:
This second sentence doesn't seem necessary for the usage of this function.
If somebody is interested this can be observed by just reading the source of the function.